beancount_import/matching.py

```diff
@@ -231,6 +247,8 @@ def __init__(self, fuzzy_match_days: int,
         self.fuzzy_match_amount = fuzzy_match_amount
         self.is_cleared = is_cleared
         self._postings = {}  # type: Dict[DatabaseDateKey, DatabaseValues]
```

Comment:
I think we can eliminate `self._postings` since it is redundant with `self._date_currency`

Reply:
I agree. I had left it in case I missed something :) I'll follow up with a change to remove it.